pages/common/pio-project.md

```diff
@@ -0,0 +1,24 @@
+# pio project
+
+> Tool to manage PlatformIO projects.
+> More information: <https://docs.platformio.org/en/latest/core/userguide/project/index.html>.
+
+- Initialize a new PlatformIO based project:
+
+`pio project init`
+
+- Initialize a new PlatformIO based project in a specific direcdory:
+
+`pio project init --project-dir {{path/to/project_directory}}`
+
+- Initialize a new PlatformIO based project, specifying a board ID:
+
+`pio project init --board {{board_id}}`
```

Comment:
Maybe we could give some specific examples here. Perhaps in the form `{{foo|bar|...}}`?

Reply:
Sure